db/schema.rb

```diff
@@ -38,7 +38,7 @@
   create_sequence "provider_users_id_seq"
   create_sequence "provider_users_providers_id_seq"
   create_sequence "providers_id_seq"
-  create_sequence "qualifications_public_id_seq", start: 196
+  create_sequence "qualifications_public_id_seq", start: 120000
```

Comment:
Docs say the default postgresql sequence max value is 2,147,483,647 so this seems a safe starting point.

Reply:
Ok! This has already run on production with a value of 110000 or so, the value here is just what it was when it was run locally